tracking/src/main/java/org/hps/recon/tracking/SensorSetup.java

```diff
@@ -50,7 +58,26 @@ protected void process(EventHeader event) {
     public void loadFittedHits(List< LCRelation > fittedHits) {
 
         for (LCRelation fittedHit : fittedHits) { 
+
+            Boolean skipHit = false;
             RawTrackerHit rawHit = FittedRawTrackerHit.getRawTrackerHit(fittedHit);
+            
+            SiSensor sensor = (SiSensor) rawHit.getDetectorElement();
+            String name = sensor.getName();
+            
+            if (sensorNames_.size() > 0) {
```

Comment:
The loop is over all the hits, this check is on the list of sensors we want to skip. I do think sensorNames_ is potentially a confusing variable name since it is really the sensors to be skipped.